virt/virt-2-4-release-notes.adoc

```diff
@@ -24,6 +24,17 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 //CNV-5438 - svirt-support-for-cnv
 * This release features significant security enhancements. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCCs to infrastructure containers in the cluster.
 
+//CNV-4572-Guest-OS
+[id="virt-guest-os-new"]
+=== Supported guest operating systems
+{VirtProductName} guests can use the following operating systems:
+
+** Red Hat Enterprise Linux 6, 7, 8.
+** Microsoft Windows Server 2012 R2, 2016, and 2019.
+** Microsoft Windows 10.
```

Comment:
- Please change the `**`s to `*` now that you've removed the initial bullet point.
- Please add `and`: `* Red Hat Enterprise Linux 6, 7, and 8.` (parallel structure with the next line)

Reply:
@ousleyp Requesting clarification, I didn't understand the comments. Can you show how you want the bullets and the text to be displayed (the complete structure). Thanks.

Reply:
Content was updated.